src/iframe-helper.js

```diff
@@ -36,6 +36,9 @@ export function listen(iframe, typeOfMessage, callback, opt_is3P) {
   const sentinel = getSentinel_(opt_is3P);
   let unlisten;
   let listener = function(event) {
+    const data = typeof event.data == 'string'
```

Comment:
Please also check whether the first char is `{` and that `opt_is3P` is not true (This is performance critical for us).